docs/resources/application.md

```diff
@@ -64,33 +71,32 @@ resource "azuread_application" "example" {
       essential             = true
       additional_properties = ["emit_as_roles"]
     }
+
+    saml2_token {
+      name = "samlexample"
+    }
   }
 
   required_resource_access {
     resource_app_id = "00000003-0000-0000-c000-000000000000"
 
     resource_access {
-      id   = "..."
+      id   = "df021288-bdef-4463-88db-98f22de89214"
       type = "Role"
     }
 
     resource_access {
-      id   = "..."
-      type = "Scope"
-    }
-
-    resource_access {
-      id   = "..."
+      id   = "b4e74841-8e56-480b-be8b-910348b18b4c"
       type = "Scope"
     }
   }
 
   required_resource_access {
-    resource_app_id = "00000002-0000-0000-c000-000000000000"
+    resource_app_id = "c5393580-f805-4401-95e8-94b7a6ef2fc2"
```

Comment:
These are real app/role/scope IDs published by Microsoft - I thought a working example would serve better since these attributes are a frequent source of confusion. I'll add some comments indicating what they are.

Reply:
is these are common ones, is it worth a data source to give a human-friendly name for these?

Reply:
Yep! I've already documented a whole lot of app IDs and plan to make those available in a data source. The individual scope/role IDs (in the `resource_access` blocks) can actually be sourced from a `data.azuread_service_principal` - I will add an example to the docs.